### PR TITLE
Animation curve path fix

### PR DESCRIPTION
**Pull request by ellthompson**
Currently if an animation curves target node contains an empty path, the animation won't resolve. This PR will resolve those animations to the node using it's name as an identifier.

Fixes: https://forum.playcanvas.com/t/is-it-possible-to-animate-morphs/17501/7

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

**Review by mvaligursky — APPROVED** [review]
looks good but wait for @willeastcott 's approval before merge as he's in he middle of testing for the release. But I assume he might want to include this.

**Comment by willeastcott** [comment]
I'm fine to include this in `1.39.0` - but it will need to be merged this morning for that to happen. 😄

**Pull request merged**
